modules/social_features/social_event/src/Form/EventSettingsForm.php

```diff
@@ -50,6 +49,24 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       ],
     ];
 
+    $form['max_enroll'] = [
```

Comment:
I think the default should be that it is *enabled*.

Reply:
Is it best practice to provide the settings in this `social_event` form while the settings are actually for the social_event_max_enroll module?

Reply:
@jaapjan @bramtenhove In acceptance criteria of https://jira.goalgorilla.com/browse/ECI-851 it was said that **setting should be set off by default**, but I agree that it doesn't make sense, because if you enable this optional module, probably you also want to have this feature enabled. So I will make it enabled and add to the config file.
Another question is where to have this config. Usually, we have it in a separate file, but I thought that we have to much settings page, and maybe it makes sense to combine all events settings into one. But I think it needs refactoring in all other modules, so I will create separate config and page for now.